tools/changelogger/src/render.rs

```diff
@@ -29,7 +29,7 @@ pub const EXAMPLE_ENTRY: &str = r#"
 # [[smithy-rs]]
 # message = "Fix typos in module documentation for generated crates"
 # references = ["smithy-rs#920"]
-# meta = { "breaking" = false, "tada" = false, "bug" = false }
+# meta = { "breaking" = false, "tada" = false, "bug" = false, "sdk" = "client | server | both" }
```

Comment:
Should we call it a `target` rather than `sdk` so that it's consistent with `CodegenTarget` in the code generator?

Reply:
Oh yes it makes more sense to use target instead of sdk.